cabal.project

```diff
@@ -48,7 +50,11 @@ packages:
   cardano-db-sync/cardano-db-sync
   cardano-db-sync/cardano-db-sync-extended
 
-index-state: 2020-07-14T22:50:55Z
+  cardano-wallet/lib/*/*.cabal
+  cardano-addresses/*/*.cabal
+  cardano-transactions
```

Comment:
It is supposed to be the other way around. cardano-addresses, cardano-wallet & cardano-transactions are using cardano-haskell as a snapshot reference, so they can't be in there.

Reply:
This PR is for adding our stuff to the `cardano-haskell/cabal.project` mega-project. It's a big list of repos which you can (try to) build using a single `cabal build all` command.  

The stack snapshots are different thing. And yes, you are correct that `cardano-{addresses,wallet,transactions}` should not be listed in the stack snapshots.